docs/source/setup/introduction_to_vrp.rst

```diff
@@ -0,0 +1,70 @@
+A brief introduction to VRP
+===========================
+
+The vehicle routing problem (VRP) is one of the most studied problems in the field of operations research motivated by applications such as postal delivery, waste collection, and many more.
+The overarching goal of the VRP is to determine a set of vehicle routes to fulfill all (or some) transportation requests at the lowest possible cost.
+
+Motivated by the enormous potential for cost savings, creating algorithms to compute cost-efficient solutions has been a primary focus for VRP researchers.
+While finding a feasible solution for a given VRP is often relatively simple, obtaining the optimal solution can be considerably more complex as most VRP variants are classified as `NP-hard <https://en.wikipedia.org/wiki/NP-hardness>`_.
+Various heuristics, metaheuristics, and exact methods have been developed to tackle the VRPs, including but not limited to, local search, genetic algorithms, and branch-and-cut algorithms.
+
+.. note::
+
+    PyVRP primarily implements heuristic and metaheuristic algorithms for solving vehicle routing problems (VRPs).
+    As these algorithms do not guarantee optimal solutions, we rigorously :doc:`benchmark <benchmarks>` them to evaluate their effectiveness.
+    For an exact state-of-the-art solver, see, for example, `VRPSolverEasy <https://github.com/inria-UFF/VRPSolverEasy>`_.
+
+
+Supported VRP variants
+----------------------
+
+In this section, we introduce the VRP variants that are supported by PyVRP.
+An extensive list of VRP variants can be found in `Toth and Vigo (2014) <https://doi.org/10.1137/1.9780898718515>`_.
+
+.. note::
+
+    Interested in variants that are not supported yet by PyVRP? Feel free to make an issue on `GitHub <https://github.com/PyVRP/PyVRP/issues>`_!
```

Comment:
I found this note somewhat distracting for an "intro to VRP" page. Maybe not mention this?